README.md

```diff
@@ -97,7 +97,7 @@ With valid `source` options as such:
 ```bash
 export DIVE_VERSION=$(curl -sL "https://api.github.com/repos/wagoodman/dive/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
 curl -OL https://github.com/wagoodman/dive/releases/download/v${DIVE_VERSION}/dive_${DIVE_VERSION}_linux_amd64.deb
-sudo apt install ./dive_${DIVE_VERSION}_linux_amd64.deb
+sudo dokg -i ./dive_${DIVE_VERSION}_linux_amd64.deb
```

Comment:
```suggestion
sudo dpkg -i ./dive_${DIVE_VERSION}_linux_amd64.deb
```

Assuming `apt install` doesn't work as expected, there's still a typo in here that needs to be fixed.